doc/rtd/topics/analyze.rst

```diff
@@ -292,10 +302,19 @@ commands below will gather the UserspaceTimestamp and InactiveExitTimestamp:
 
 .. code-block:: shell-session
 
-  $ systemctl show -p UserspaceTimestampMonotonic
-  UserspaceTimestampMonotonic=989279
-  $ systemctl show cloud-init-local -p InactiveExitTimestampMonotonic
-  InactiveExitTimestampMonotonic=4493126
+   $ systemctl show -p UserspaceTimestampMonotonic
+
```

Comment:
Same comment about `Example Output:` line between command and example output.

Reply:
+1 - sorry for making you write the same thing over and over :D